lib/datacrafts_io_skeleton/composer.rb

```diff
@@ -0,0 +1,34 @@
+require "thor"
+require_relative "./version"
+require_relative "./rails_creator"
+
+module DatacraftsIoSkeleton
+  # Defines all CLI commands, their descriptions, params and aliases.
+  class Composer < Thor
+    desc "create APP_NAME", "This will create new rails app."
+    long_desc <<-CREATE_NEW_APP
+     This command will create new rails app with configured rails_helper.rb and .rubocop.yml files. 
+     All major gems are also included.
```

Comment:
Description will be more clear for nondatacrafts developers if we put this text in the Readme file right after the message `Create new apps quickly!`.